.github/workflows/code_checks.yml

```diff
@@ -139,6 +115,16 @@ jobs:
           docker compose -f docker-compose.test.yml run web bash \
           -c "CI=true DISABLE_BOOTSNAP=true bundle exec parallel_rspec spec/ modules/ -n 24 -o '--color --tty'"
 
+      - name: Set Test Status
+        id: test-status
+        if: always()
+        run: |
+          if [ "${{ job.status }}" = "success" ]; then
+            echo "status=success" >> $GITHUB_OUTPUT
+          else
+            echo "status=failure" >> $GITHUB_OUTPUT
+          fi
```

Comment:
By capturing the result, we can make sure only one of the two labels gets set at a time